src/commands/run.js

```diff
@@ -1,17 +1,21 @@
 const { ApifyCommand } = require('../lib/apify_command');
 const execWithLog = require('../lib/exec');
-const { LOCAL_ENV_VARS, PROXY_PASSWORD_ENV_VAR } = require('../lib/consts');
+const { LOCAL_ENV_VARS } = require('../lib/consts');
+const { ENV_VARS } = require('apify-shared/consts');
 const { getLocalUserInfo } = require('../lib/utils');
 
 // TODO: Shall we also pass APIFY_TOKEN ?
 
 class RunCommand extends ApifyCommand {
     static async run() {
-        const { proxy } = getLocalUserInfo();
-        const env = Object.assign(process.env, LOCAL_ENV_VARS);
-        if (proxy && proxy.password) {
-            env[PROXY_PASSWORD_ENV_VAR] = proxy.password;
-        }
+        const { proxy, id: userId, token } = getLocalUserInfo();
+
+        // NOTE: User can overwrite env vars
+        const env = Object.assign(LOCAL_ENV_VARS, process.env);
+
+        if (proxy && proxy.password) env[ENV_VARS.PROXY_PASSWORD] = proxy.password;
```

Comment:
If user passes PROXY_PASSWORD in env vars, then we probably shouldn't overwrite it here. Env vars should have priority over everything...

Reply:
and same for user_id and token